infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -101,6 +101,9 @@ param parExcludedPolicyAssignments array = []
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry')
 param parTelemetryOptOut bool = false
 
+@sys.description('Effect type for sovereign policy definitions')
+param parSovereigntyPolicyEffect string = 'Deny'
```

Comment:
Can we please add this params `parPolicyAssignmentSovereigntyConfidential` and `parTopLevelPolicyAssignmentSovereigntyGlobal` user-defined type objects as properties and only allow valid policy effects? I think it is a bit cleaner to keep the sovereign stuff intertwined as much as possible and then also gives the ability to set different effects per assignment.